src/scenes/Application/components/ApplicationForm/ApplicationFormSteps.tsx

```diff
@@ -40,55 +43,64 @@ const ApplicationFormSteps = async ({
         hackathonId={hackathonId}
       />
       <Stack className="w-full" direction="column">
-        {data && (
-          <>
-            <Heading className="mx-auto md:mb-6">
-              Your application for {data.hackathonName}:
-            </Heading>
-            <Stack direction="column" className="gap-5 md:gap-20 w-full">
-              <ApplicationStatusCard status={data.application.status} />
-              {data.application.status === "open" && (
-                <div className="mx-auto">
-                  <Stack
-                    justify="around"
-                    spaceAfter="medium"
-                    className="md:flex-row flex-col w-full md:gap-10"
-                  >
-                    {data.steps.map((step) => (
+        {data &&
+          (closedPortal && data.application.status === "open" ? (
+            <Card className="p-5 pt-10 mx-auto container bg-primaryTitle">
+              <Heading className="text-white">
+                Thank you for your interest, but the applications for this year
+                are already closed. Please sign up for a hackathon next year.
+              </Heading>
+            </Card>
```

Comment:
```suggestion
            <Card className="p-5 w-fit mx-auto container bg-primaryTitle">
              <Text className="text-white w-full text-center text-lg">
                Thank you for your interest, but the applications for this year
                are already closed. Please sign up for a hackathon next year.
              </Text>
            </Card>
```
I think something like this looks much better here